index.html

```diff
@@ -103,46 +113,224 @@
+                        <li>
+                            <a href="#">RichField Chairs</a>
+                        </li>
+                        <li>
+                            <a href="#">Delivery in 24 hours</a>
+                        </li>
+                        <li>
+                            <a href="#">Free shipping</a>
+                        </li>
+                        <li>
+                            <a href="#">Our Guarantee</a>
+                        </li>
+                        <li>
+                            <a href="#">Email Preferences</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="column-two col-md-4">
+                    <h6>LET US HELP</h6>
+                    <ul>
+                        <li>
+                            <a href="#">Help Center</a>
+                        </li>
+                        <li>
+                            <a href="#">Contact Customer Care</a>
+                        </li>
+                        <li>
+                            <a href="#">Return Policy</a>
+                        </li>
+                        <li>
+                            <a href="#">Interational Help</a>
+                        </li>
+                        <li>
+                            <a href="#">Accessibility</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="column-three col-md-4">
+                    <h6>MORE WAYS TO SHOP</h6>
+                    <ul>
+                        <li>
+                            <a href="#">Tips & Ideas</a>
+                        </li>
+                        <li>
+                            <a href="#">Coupons</a>
+                        </li>
+                        <li>
+                            <a href="#">Deals</a>
+                        </li>
+                        <li>
+                            <a href="#">Clearance</a>
+                        </li>
+                        <li>
+                            <a href="#">New Arrivals</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+    <script src="https://code.jquery.com/jquery-2.2.0.min.js" type="text/javascript"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
+    <script type="text/javascript" src="js/main.js"></script>
```

Comment:
Move these to top of the document

Reply:
done

Reply:
When I moved this to the top, the slick didn't work.